doc/releasenotes.md

```diff
@@ -59,4 +59,11 @@
 
 ## Aerospike Loader [2.4.3] Release Date [4 Jan 2022]
 ### Fix/Changes:
--   Update log4j to version 2.17.1.
\ No newline at end of file
+-   Update log4j to version 2.17.1.
+
+## Aerospike Loader [3.0.0] Release Date [27 Jan 2023]
+### Breaking Changes:
+-   Upgrade Aerosppike Java client to 6.1.6 which is only compatible with Aerospike server 4.9 or newer.
+### New features:
+-   Write key ordered maps by default.
```

Comment:
```suggestion
-   Upgrade Aerosppike Java client to 6.1.6 which is only compatible with Aerospike server 4.9 or newer.
-   Write key ordered maps by default.
### New features:
```

Reply:
Fixed a typo that caused this to go out of date (unmergable) but did make the change.